pybsn/__init__.py

```diff
@@ -423,7 +565,9 @@ def _attempt_modern_login(session, url, username, password):
         response.raise_for_status()
 
 
-def connect(host, username=None, password=None, token=None, login=None, verify_tls=False, session_headers=None):
+def connect(host, username=None, password=None, token=None, login=None,
+            verify_tls=False, session_headers=None,
+            timeout=None):
```

Comment:
connect is intentionally None timeout here?

Reply:
Yes, for backwards compatibility